src/proposals/proposalTypes/GovernanceProposal.sol

```diff
@@ -0,0 +1,141 @@
+pragma solidity 0.8.19;
+
+import {Test} from "@forge-std/Test.sol";
+import {console} from "@forge-std/console.sol";
+
+import {Addresses} from "@proposals/Addresses.sol";
+import {Well} from "@protocol/Governance/deprecated/Well.sol";
+import {Proposal} from "@proposals/proposalTypes/Proposal.sol";
+import {MoonwellArtemisGovernor, TimelockInterface} from "@protocol/Governance/deprecated/MoonwellArtemisGovernor.sol";
+
+abstract contract GovernanceProposal is Proposal {
+    bool private DEBUG;
+
+    struct GovernanceAction {
+        address target;
+        uint256 value;
+        string description;
+        bytes data;
+    }
+
+    GovernanceAction[] public actions;
+
+    /// @notice set the debug flag
+    function setDebug(bool debug) public {
+        DEBUG = debug;
+    }
+
+    function printCalldata(Addresses addresses) public override {}
+
+    function printProposalActionSteps() public override {}
+
+    /// @notice deal tokens
+    /// @param token address of the token
+    /// @param toAddress address to send tokens to
+    /// @param amount amount to deal
+    function _deal(address token, address toAddress, uint256 amount) internal {
+        deal(token, toAddress, amount, false);
```

Comment:
Could you please add a comment here explaining to which contract this call is being made? I couldn't find

Reply:
I thought it was supposed to go to the foundry, but shouldn't it be vm.deal? 🤔